packages/mobile/src/send/SendConfirmation.test.tsx

```diff
@@ -50,9 +50,9 @@ describe('SendConfirmation', () => {
     expect(queryByText('0.0100')).toBeNull()
 
     // Wait for fee to be calculated and displayed
-    await waitForElement(() => getByText('0.0100'))
+    // await waitForElement(() => getByText('0.0100'))
```

Comment:
@jeanregisser I tried for a while to get this to work but no luck. I believe you wrote this test, any advice?

Reply:
No, I tried a bunch of variations of the string, couldn't get it to work. I would expect it to be `$0.001`

Reply:
I remember having a similar issue at some point, which got resolved by changing the markup. My hunch is there's a bug somewhere in `react-native-testing-library` when the text is nested and prevents `getByText` from finding it.

I guess leave it out for now and add a TODO. 🤷‍♂️